CHANGELOG/CHANGELOG-1.6.0.md

```diff
@@ -60,3 +60,8 @@
 - vSphere : Can not create VM host object. ([#592](https://github.com/dell/csm/issues/592))
 - PowerScale Replication: Artifacts are not properly cleaned after deletion. ([#523](https://github.com/dell/csm/issues/523))
 - PVC fails to resize with the message spec.capacity[storage]: Invalid value: "0": must be greater than zero. ([#507](https://github.com/dell/csm/issues/507))
+
+#### Known Issues
+
+- PowerFlex driver deployment via CSM Operator should work without having the user to create ConfigMap([#713](https://github.com/dell/csm/issues/713))
+- Powerflex driver specific parameters mentioned in the samples are not configurable when deployed via CSM Operator ([#714](https://github.com/dell/csm/issues/714))
```

Comment:
Could I ask a favor: Could you also add the below line since it is not captured here

Reply:
CSM object does not track available deployment count when down scaling to n-1 , where n is number of nodes.